internal/backend/foreground_windows.go

```diff
@@ -2,12 +2,15 @@ package backend
 
 import (
 	"os/exec"
+	"syscall"
 
 	"github.com/restic/restic/internal/errors"
 )
 
 func startForeground(cmd *exec.Cmd) (bg func() error, err error) {
 	// just start the process and hope for the best
+	cmd.SysProcAttr = &syscall.SysProcAttr{}
+	cmd.SysProcAttr.CreationFlags = 0x00000008 // DETACHED_PROCESS
```

Comment:
```suggestion
	cmd.SysProcAttr.CreationFlags = windows.DETACHED_PROCESS
```

(from golang.org/x/sys/windows).